src/main/joyride/nrepl.cljs

```diff
@@ -65,6 +66,23 @@
             "ops" (zipmap (map name (keys ops)) (repeat {}))
             "status" ["done"]}))
 
+(def pretty-print-fns-map
+  {"clojure.core/prn" prn
+   "clojure.pprint/pprint" pp/pprint
```

Comment:
Being `.cljs` I find myself using `cljs.pprint/pprint` often, could we ~~sneak that in~~ add that here as well?

Reply:
Yes

Reply:
Help me understand. Does the `clojure.pprint` namespace get available to joyride scripts from this? Testing the VSIX and trying to require it, I get:

```
; ENOENT: no such file or directory, open '/Users/pez/.config/joyride/scripts/.joyride/scripts/clojure/pprint.cljs'
```

Reply:
No it doesn't. We need to add that in the SCI config. But clojure.pprint does get used here to print values sent back the the nREPL client.

Reply:
So then the addition of `cljs.pprint` is for someone who configures the nrepl printer?

Reply:
Yes, but by default CIDER uses a cider-specific function. We just map that to cljs.pprint.